src/logcollector/read_win_el.c

```diff
@@ -283,6 +283,7 @@ void readel(os_el *el, int printit)
     int size_left;
     int str_size;
     int id;
+    static int counter = 0;
```

Comment:
Logcollector is multithreading so using a static variable here is thread-unsafe unless it is protected by mutexes when writing and reading it. Let's protect that variable or look for another way to show the log just once.

Reply:
It is affecting to eventchannel only, the eventlog reading is protected with a mutex.